src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.html

```diff
@@ -0,0 +1,41 @@
+<div *ngIf="contentData">
```

Comment:
I would suggest you to use the following code. Just need to implement the TODOs.
```html
<div class="row">
  <div class="col-md-12">
    <tabset>
      <tab heading="Cluster log"
           class="text-monospace"
           i18n-heading>
        <div class="well">
           <!-- TODO: add a <cd-info-card *ngIf="!contentData?.clog?.length"> here, informing the user there are no logs available -->
          <span *ngFor="let line of contentData?.clog">
            {{ line.stamp }}&nbsp;{{ line.priority }}&nbsp;
            <span [ngStyle]="line">
              {{ line.message }}
              <br>
            </span>
          </span>
        </div>
      </tab>
      <tab heading="Audit log"
           class="text-monospace"
           i18n-heading>
        <div class="well">
            <!-- TODO: add a <cd-info-card *ngIf="!contentData?.audit_log?.length"> here, informing the user there are no logs available -->
          <span *ngFor="let line of contentData?.audit_log">
            {{ line.stamp }}&nbsp;{{ line.priority }}&nbsp;
            <span [ngStyle]="line"
                  style="font-weight: bold;">
              {{ line.message }}
              <br>
            </span>
          </span>
        </div>
      </tab>
    </tabset>
  </div>
</div>

```